Lib/asyncio/base_events.py

```diff
@@ -200,6 +200,17 @@ def _set_nodelay(sock):
         pass
 
 
+if hasattr(socket, 'TCP_QUICKACK'):
+    def _set_quickack(sock, val):
```

Comment:
Is this used anywhere but in the test? Why should we add tests for something that isn't used?

Reply:
The property is already available for all users of socket, it just didn't have a test before and was only available on Linux.  In addition to adding support for the property on windows platform, it's now actually being tested.

Reply:
This is a new private function in a public module, not a test. If we don't need the private function, we don't want it. It looks like every other reviewer has said the same thing, so you probably just want to either delete this function or move it into the test module.

Reply:
I realized this after commenting, will move.